aptos-move/framework/aptos-token-objects/sources/token.move

```diff
@@ -1012,29 +969,17 @@ module aptos_token_objects::token {
         assert!(!object::is_object(token_addr), 2);
     }
 
-    #[test(fx = @aptos_framework, creator = @0x123)]
-    fun test_upgrade_to_concurrent_and_numbered_tokens(fx: &signer, creator: &signer) acquires Token, TokenIdentifiers {
-        use std::debug;
-
-        let feature = features::get_concurrent_token_v2_feature();
-        let agg_feature = features::get_aggregator_v2_api_feature();
-        let auid_feature = features::get_auids();
-        let module_event_feature = features::get_module_event_feature();
```

Comment:
module event feature also cannot be reverted?

Reply:
it can be reverted, but default is true - so we don't need to enable it in this test.

though that can potentially also be cleaned up @lightmark